documentation/book/assembly-storage.adoc

```diff
@@ -0,0 +1,22 @@
+// This assembly is included in the following assemblies:
+//
+// cluster-operator.adoc
+
+// Save the context of the assembly that is including this one.
+// This is necessary for including assemblies in assemblies.
+// See also the complementary step on the last line of this file.
+:parent-context: {context}
+
+[id='assembly-storage-{context}']
+= Kafka and Zookeeper storage
+
+Kafka brokers and Zookeeper are stateful applications.
+They need to store data on disks.
+{ProductName} allows users to configure the type of storage which they want to use for Kafka and Zookeeper.
+Storage configuration is mandatory and has to be specified in every `Kafka` resource.
```

Comment:
add comma before which (storage, which they) and second person usage:
{ProductName} allows you to configure the type of storage for Kafka and Zookeeper.